jaeger_client/span.py

```diff
@@ -37,6 +40,7 @@ def __init__(self, context, tracer, operation_name,
         self.operation_name = operation_name
         self.start_time = start_time or time.time()
         self.end_time = None
+        self.finished = False
```

Comment:
please add `finished` to `__slots__` above

Reply:
Oh, didn't notice that. Done!